### ISSUE TITLE
Code not working

### ISSUE THREAD

**Issue by ruarkvallen** [WPManageNinja/fluentform-snippets#1]
Added the code as is via Code Snippets, but form still accepts gmail and goes to our thank you page and adds the entries to the database. Is there anything i missed?

**Comment by LauGau** [comment]
Yes you forget the PHP part in your function.php file like this :

```php
add_filter('fluentform_validate_input_item_input_email',
    function($errorMessage, $field, $formData, $fields, $form) {
        $target_form_id = 1;
        if($form->id != $target_form_id ) { 
            return $errorMessage ; 
        }

        if( $formData['email'] !=  $formData["email_1"]  ){
            $errorMessage=  'The emails doesn't match.';
        }

        return [$errorMessage];
    },
10, 5);

```

**Comment by AkhilAniyan** [comment]
Still the code not working!

Even though the emails are correct, it's not submitting! See here: https://prnt.sc/vc6158

If the email is wrong, the error shows fine: https://prnt.sc/vc61xb

Code Added to function.php: https://prnt.sc/vc63wr

**Comment by AkhilAniyan** [comment]
@techjewel Can you please help?

**Comment by alex-authlab** [comment]
Try this  : 
<pre>
add_filter('fluentform_validate_input_item_input_email', function($errorMessage, $field, $formData, $fields, $form) {
    $target_form_id = 8;
    if($form->id != $target_form_id ) {
     return $errorMessage ;
    }
    
    if( $formData['email'] !=  $formData['email_1']  ){
        $errorMessage=  ['Error! Email does not match'];
    }
    return $errorMessage;
}, 10, 5);
</pre>
If it does not work can you share your form URL link?
